index.bs

```diff
@@ -902,10 +904,10 @@ partial interface HTMLIFrameElement {
     Given a feature (|feature|), an <a>origin</a> (|origin|), and a <a>browsing
     context</a> (|browsingContext|), this algorithm returns the <a>inherited
     policy</a> for that feature.
-    1. If |browsingContext| is the [=nested browsing context=] of a [=browsing
-      context container=], return the result of executing <a abstract-op>Define
-      an inherited policy for feature in container at origin</a> for |feature|
-      in |browsingContext|'s <a>browsing context container</a> at |origin|.
+    1. If |browsingContext| is the [=nested navigable=] of a [=navigable
```

Comment:
Here you're equating browsing contexts and navigables, which is not great. Ideally the algorithm should be updated to only operate on navigables.

Reply:
I've updated the interface with HTML now; we don't actually need browsing context to be passed in at all. I couldn't use navigable here previously, because these algorithms are called from https://html.spec.whatwg.org/#creating-a-new-browsing-context, before a navigable is created. However, we do have *embedder* at that point, which is what was actually being used.

*embedder* is currently just an Element; I'm pretty sure it's (currently) always going to be a navigable container, but I put an assertion in, so that if that ever changes, we know that this needs to be revisited.